fbw-a32nx/src/localization/flypad/en.json

```
@@ -81,6 +81,7 @@
       "CostIndex": "Cost Index",
       "CruiseAlt": "Cruise Alt.",
       "ImportSimBriefData": "Import SimBrief Data",
+      "ToastFuelPayloadImported": "Fuel & Payload imported!",
```

```suggestion
      "ToastFuelPayloadImported": "Fuel & Payload imported!",
```
Have these strings been added to localezy? Currently this is a manual process, if you were not aware for future reference.

For Mav / Localezy Maintainers: If these keys do not already exist, please added to localezy for this to show up in a production build.

---

> No was not added - I don't know how :/ Sorry!
> 
> On that Matter: would a changed Text also need to be added manually? Cause my former PR 8407 included something like that (Key: GSXFuelSyncEnabled).

Text changes when the key remains the same are best done in Localazy.
Key changes are best done with simply a new key. There is new feature to change even existing keys to not loose translations - ping me if required.